packages/gatsby-theme-patternfly-org/components/cssVariables/cssVariables.js

```diff
@@ -1,105 +1,211 @@
+  <div>
+    <div
+      style={{
+        padding: `4px 0 4px calc(${global_spacer_md.value})`
+      }}
+    >
+      <span style={{ paddingLeft: '16px' }}>
+        {property}
+      </span>
+    </div>
+    {values.map((entry, index) => (
+      <div
+        style={{
+          padding: `4px 0 4px calc(${global_spacer_md.value} * ${index + 3})`
+        }}
+      >
+        <LevelUpAltIcon style={{ transform: 'rotate(90deg)' }} />
+        <span style={{ paddingLeft: '16px' }}>
+          {entry}
+        </span>
+      </div>
+    ))}
+  </div>
+);
+
 export class CSSVariables extends React.Component {
   constructor(props) {
     super(props);
     // Ensure array in case of multiple prefixes
-    this.prefix = typeof props.prefix === 'string'
-      ? [props.prefix]
-      : props.prefix;
-    const initialRows = Object.entries(tokensModule)
-      .filter(([_key, val]) => {
+    this.prefix =
+      typeof props.prefix === "string" ? [props.prefix] : props.prefix;
+    const applicableFiles = Object.entries(tokensModule)
+      .filter(([key, val]) => {
         for (let i = 0; i < this.prefix.length; i++) {
-          if (val.name && val.name.includes(this.prefix[i])) {
+          if (
+            key === this.prefix[i].replace("pf-", "").replace(/-+/g, "_")
+          ) {
             return true;
           }
         }
-        return false
+        return false;
       })
       .sort(([key1], [key2]) => key1.localeCompare(key2))
-      .map(([key, val]) => [
-        val.name,
-        key,
-        val.value
-      ]);
+      .map(([key, val]) => {
+        if (props.selector) {
+          return {
+            [props.selector]: val[props.selector]
+          }
+        }
+        return val;
+      });
 
-    this.columns = [
-      { title: 'Variable', transforms: [sortable] },
-      { title: 'React Token', transforms: [sortable] },
-      { title: 'Value', transforms: [sortable] }
+    this.columns = props.hideSelectorColumn ? [] : [
+      {
+        title: "Selector",
+        transforms: [sortable],
+        cellFormatters: [expandable]
+      }
     ];
-    
+    this.columns = this.columns.concat([
+      { title: "Variable", transforms: [sortable] },
+      { title: "React Token", transforms: [sortable] },
+      { title: "Value", transforms: [sortable] }
+    ]);
+
     this.state = {
-      filterValue: '',
-      rows: initialRows,
+      applicableFiles,
+      rows: this.getFilteredRows(applicableFiles),
       sortBy: {
         index: 0,
-        direction: 'asc' // a-z
+        direction: "asc" // a-z
       }
     };
+
+    this.getFilteredRows = this.getFilteredRows.bind(this);
+    this.getDebouncedFiltedRows = this.getDebouncedFiltedRows.bind(this);
+    this.onCollapse = this.onCollapse.bind(this);
   }
 
-  onFilterChange = (_change, event) => {
+  getFilteredRows = (applicableFiles, searchRE) => {
+    let filteredRows = [];
+    let rowNumber = -1;
+    applicableFiles.forEach(file => {
```

Comment:
There's a much prettier way to write this loop with `applicableFiles.flatMap(file => Object.entries(file).filter(searchRE).map(val => filteredRow)` if you're feeling up for it